modules/overrides/limits.go

```diff
@@ -70,6 +70,9 @@ type Limits struct {
 	// Querier enforced limits.
 	MaxBytesPerTagValuesQuery int `yaml:"max_bytes_per_tag_values_query" json:"max_bytes_per_tag_values_query"`
 
+	// QueryFrontend enforced limits
+	MaxSearchDuration model.Duration `yaml:"max_search_duration" json:"max_search_duration"`
```

Comment:
Is there a reason for `model.Duration`?  Just comparing to `MetricsGeneratorCollectionInterval time.Duration` above.